extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -465,12 +465,19 @@ Pass the action object to your front end. The Adyen web component uses this to h
 
 > See [Adyen documentation](https://docs.adyen.com/online-payments/web-components/integrated-before-5-0-0#step-4-additional-front-end) for more information how to perform additional front end actions.
 
-### Klarna payment and Affirm payment
+### Adding invoice lines (lineItems) to request
 
-For Klarna payment and Affirm payment, it is necessary to provide [line item details](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/latest/payments__reqParam_lineItems) in `makePaymentRequest`.
-The extension module can add the line item details for you if [the payment is added to a cart](https://docs.commercetools.com/http-api-projects-carts#add-payment).
+For some payment methods (such as Klarna & Affirm), it is necessary to provide [line item details](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/latest/payments__reqParam_lineItems) in `makePaymentRequest`.
 
-Using Adyen Web Components, create `makePaymentRequest` **WITHOUT** `lineItems` attribute.
+The extension module can add the line item details for you, but you might need to do following steps:
+
+- The commercetools payment [referenced in the commercetools cart](https://docs.commercetools.com/http-api-projects-carts#add-payment).
+- To use commercetools lineItems, do not include `lineItems` field in the `makePaymentRequest`, otherwise the extension module will use provided lineItems already passed by `makePaymentRequest`.
+- Configuration set, there are 2 options:
+  1. `addCommercetoolsLineItems` flag set `true` on your extension [config](./HowToRun.md#other-configurations). If set to **true**, it would add lineItems to payment methods that requires lineItems.
+  1. `addCommercetoolsLineItems` field added to `makePaymentRequest`.
```

Comment:
Maybe an example of this could be useful?

Reply:
What about this: https://github.com/commercetools/commercetools-adyen-integration/pull/918/commits/bd2df26c0737c3660348c71340aa904006200f31